### PR TITLE
Add new logos

### PR DESCRIPTION
**Pull request by soerenmartius**
# Changelog

This PR adds new icons as well as versions with white fonts of our existing logo. It also deletes the outdated `vertical-logo-small-font.svg` logo.

We should only merge this once we updated the `README.md`s in all public repositories.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,10 +5,24 @@ our logo, content and trademarks, without having to negotiate legal agreements
...
 
-<img src="./mineiros-vertical-logo.svg" width="300" />
+<img src="./mineiros-primary-logo-white-font.svg" width="400" />
 
-<img src="./mineiros-vertical-logo-larger-font.svg" width="300" />
+## Vertical Logos
+
+<img src="./mineiros-vertical-logo.svg" width="400" />
+
+<img src="./mineiros-vertical-logo-white-font.svg width="400" />
```
Comment (thenaturalist) [inline README.md:18]:
```suggestion
<img src="./mineiros-vertical-logo-white-font.svg" width="400" />
```

Comment (soerenmartius) [reply README.md:18]: good catch!

**Comment by thenaturalist** [comment]
lgtm

File: mineiros-vertical-logo-white-font.svg [hunk 2]
```diff
@@ -9,7 +9,7 @@
         isolation: isolate;
```
Comment (mariux) [inline mineiros-vertical-logo-white-font.svg:9]: can not be renamed is used in public for ever.. ;)

**Pull request merged**